sale_order_line_date/models/sale_order_line.py

```diff
@@ -0,0 +1,39 @@
+# © 2016 OdooMRP team
+# © 2016 AvanzOSC
+# © 2016 Serv. Tecnol. Avanzados - Pedro M. Baeza
+# © 2016 Eficent Business and IT Consulting Services, S.L.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = 'sale.order.line'
+
+    commitment_date = fields.Datetime()
```

Comment:
Shouldn't have migration script as you changed the field ?

Reply:
As we never used this module, we just migrated what we needed.
Maybe add the attributes "old_name='requested_date'" would be enough, but we won't do the retro-compatibility fix. Sorry

Reply:
Yes, at least the old_name should be sufficent. Could you add it ?

Reply:
It is done.
As told before, we tried to help the OCA, especially for the switzerland modules.
The process change <--> review does not apply only once but can last very long.
For that we are an enterprise, we can't spend a lot of time with reviews.

Reply:
Thank you.

> The process change <--> review does not apply only once but can last very long.
> For that we are an enterprise, we can't spend a lot of time with reviews.

That's a point of view/ Most of people in OCA are tied to enterprise too. IMHO code review should be part of development as you can confont to other people, get ideas, stay in touch with changes, ... many advantages. But that depends on enterprise philosophy, that's true.